.travis.yml

```diff
@@ -34,13 +34,21 @@ matrix:
         - cargo clippy --all --all-targets -- -D warnings
         - cargo fmt --all -- --check
         # Build and publish compact image with compiled binary
-        - docker build --tag stablex-binary --build-arg SOLVER_BASE=163030813197.dkr.ecr.us-east-1.amazonaws.com/dex-solver:v0.6.0 --build-arg RUST_BASE=rust-binary -f docker/rust/Dockerfile .
-        # StableX e2e Tests (Ganache)
-        - docker-compose -f docker-compose.yml -f docker-compose.binary.yml up -d stablex
+        - docker build --tag stablex-binary-public --build-arg RUST_BASE=rust-binary -f docker/rust/Dockerfile .
+        # StableX e2e Tests (Ganache) - open solver
+        - docker-compose -f docker-compose.yml -f docker-compose.open-solver.yml -f docker-compose.binary.public.yml  up -d stablex
         - cargo test -p e2e ganache -- --nocapture
         - docker-compose logs
+        # StableX e2e Tests (Ganache) - naive solver
```

Comment:
Do we really want to keep the naive solver test? Not sure this is required.

Reply:
I would say we either keep the test or cut out the naive solver completely.

Reply:
Good point. I think we should keep it as it's the easiest way to run the entire system without any downloading any additional image